data/sql_setup/prepare_schedule_d.sql

```diff
@@ -0,0 +1,19 @@
+-- Create trigger to maintain Schedule C search columns
+ALTER TABLE disclosure.nml_sched_d DROP COLUMN if EXISTS creditor_debtor_name_text;
+
+ALTER TABLE disclosure.nml_sched_d add column creditor_debtor_name_text tsvector;
```

Comment:
I think it would be safe, because the column is built from another column on that table that will still exist.  That single statement may be better I'm not sure.